docs/6-filesStructure.md

```diff
@@ -24,3 +24,176 @@
 > │   └── subComponentFolder (if has)
 > │       └── ...
 > ```
+
+## Creating New Routes
+
+The application uses Redux First Router for routing. To create a new route, follow these steps:
+
+### 1. Define Route Constants
+
+Add your route constant to `/src/controllers/pages/constants.js`:
+
+```javascript
+export const MY_NEW_PAGE = 'MY_NEW_PAGE';
+```
+
+Add it to the `pageNames` object in the same file:
+
+```javascript
+export const pageNames = {
+  // ... existing pages
+  MY_NEW_PAGE,
+};
+```
+
+### 2. Export Route Constant
+
+Add the route constant to the exports in `/src/controllers/pages/index.js`:
+
+```javascript
+export {
+  // ... existing exports
+  MY_NEW_PAGE,
+} from './constants';
+```
+
+### 3. Create Page Component
+
+Create your page component in `/src/pages/inside/myNewPage/`:
+
+```javascript
+// myNewPage.jsx
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { SettingsLayout } from 'layouts/settingsLayout';
+
+export const MyNewPage = () => {
+  const { formatMessage } = useIntl();
```

Comment:
```suggestion
```typescript
// myNewPage.tsx
import { FC } from 'react';
import { useIntl } from 'react-intl';
import { SettingsLayout } from 'layouts/settingsLayout';

export const MyNewPage: FC = () => {
  const { formatMessage } = useIntl();
```

Reply:
I see no reason to enforce `FC` since it has no value. Typescript React works perfectly with typed functions.
Good part is that `FC` with typescript 5.1 is not considered harmful anymore, but let's have an explicit decision [whether we want to enforce it](https://www.totaltypescript.com/you-can-stop-hating-react-fc)?

Reply:
done